plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleHttpServerStateInterceptor.java

```diff
@@ -34,7 +35,7 @@ public HttpServerHandleHttpServerStateInterceptor(TraceContext traceContext, Met
     }
 
     public boolean validate(Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```

Comment:
I think `args.length < 2` would be better to change `ArrayUtils.getLength(args) < 2`, args could be 0 size arrays.

Reply:
```java
      if (!(args[1] instanceof HttpResponse)) {
            // Skip LastHttpContent or else.
            return false;
        }
```
If you look at the meaning of the whole code, `getLength(args) != 1` seems to be more accurate.